garnbarn_api/services/notification/send_notification.py

```diff
@@ -0,0 +1,17 @@
+from django.dispatch.dispatcher import receiver
+from garnbarn_api.services.notification.platforms.notification_platform import NotificationPlatform
+from garnbarn_api.services.notification.platforms.line import Line
```

Comment:
This line is unnecessary

Reply:
.__subclasses__ won't work if we don't import the classes.